packages/flutter_tools/lib/src/dart/package_map.dart

```diff
@@ -25,10 +25,10 @@ Future<PackageConfig> loadPackageConfigWithLogging(File file, {
     file.absolute.uri,
     loader: (Uri uri) async {
       final File configFile = fileSystem.file(uri);
-      if (!configFile.existsSync()) {
+      if (!await configFile.exists()) {
         return null;
       }
-      return Future<Uint8List>.value(configFile.readAsBytesSync());
+      return Future<Uint8List>.value(await configFile.readAsBytes());
```

Comment:
We don't need an await here, right?

```suggestion
      return configFile.readAsBytes();
```

Reply:
I had to revert this, for details see https://github.com/flutter/flutter/issues/134621.